common/djangoapps/util/enterprise_helpers.py

```diff
@@ -225,3 +330,37 @@ def get_enterprise_branding_filter_param(request):
 
     """
     return request.session.get(ENTERPRISE_CUSTOMER_BRANDING_OVERRIDE_DETAILS, None)
+
+
+def get_cache_key(**kwargs):
```

Comment:
Thanks for adding this -- is there one already available in a utils package or library that we could use so that the key algorithm matches what is used in other LMS caches?